capdl-loader-app/src/main.c

```diff
@@ -1536,6 +1536,7 @@ static void init_level_3(CDL_Model *spec, CDL_ObjID level_0_obj, uintptr_t level
     }
 }
 
+#if (CDL_PT_NUM_LEVELS >= 2)
```

Comment:
Seems this could be moved up as `init_level_3()` is only called byte `init_level_2()`.

Reply:
Currently every definition of CDL_PT_NUM_LEVELS is >= 2.  I placed this in to indicate the structure of the code if anything is added in the future that only has 1 PT levels.